content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -179,15 +181,31 @@ python3 -m datatrails_scitt_samples.scripts.create_hashed_signed_statement \
     ```bash
     python3 -m datatrails_scitt_samples.scripts.verify_receipt \
       --transparent-statement-file $TRANSPARENT_STATEMENT_FILE \
-      --leaf $LEAF
+      --leaf $(jq -r .leaf <<<"$RESPONSE")
     ```
 
-    Following the example above, $LEAF should be:
+    The verification should pass with:
 
     ```output
-    30f5650fbe3355ca892094a3fbe88e5fa3a9ae47fe3d0bbace348181eb2b76db
+    verification passed
     ```
 
+1. Simulate a failed verification, by altering the `.leaf` value
+
+    ```bash
+
+    python3 -m datatrails_scitt_samples.scripts.verify_receipt \
+      --transparent-statement-file $TRANSPARENT_STATEMENT_FILE \
+      --leaf $(jq -r .leaf <<<"$RESPONSE")"-foo"
+    ```
+
+    The verification should fail with:
+
+    ```output
+    ERROR:verify-receipt:failed to parse leaf hash
```

Comment:
The output for a failed verification is just "verification failed". ERROR output means the command line paramater was not accepted